lib/smart-app.js

```diff
@@ -106,11 +111,22 @@ module.exports = class SmartApp {
 		return this
 	}
 
+	/**
+	 * Manually set the SmartThings Key API host
+	 * @param {String} url The host URL
+	 * @default https://key.smartthings.com
+	 * @returns {SmartApp} SmartApp instance
+	 */
+	keyApiHost(url) {
```

Comment:
Adding this function to bring it in line with the fact that we allow users to change the API / OAuth URLs in the same fashion.